jenkins-slave/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM fedora:latest
+MAINTAINER http://fedoraproject.org/wiki/Cloud
+
+RUN dnf -y update && dnf clean all
+RUN dnf -y install java-1.8.0-openjdk openssh-server
+
+RUN ssh-keygen -A
+RUN sed -i 's|session    required     pam_loginuid.so|session    optional     pam_loginuid.so|g' /etc/pam.d/sshd
+
+RUN mkdir -p /var/run/sshd
+RUN useradd jenkins
+RUN echo "jenkins:jenkins" | chpasswd
```

Comment:
Should this be done during the docker build or through an install script before the user runs it?